sh syntax error when static build is enabled

As the title says
## option
```
./compile.sh -s
```
## error
```
./compile.sh: line 298: [: missing `]'
```
https://github.com/pmmp/php-build-scripts/blob/e90ff50310cb1b65ab5488843af1f898407ebf1a/compile.sh#L292-L302
## patch
https://github.com/pmmp/php-build-scripts/pull/143

## environment
```
ubuntu 22.04.1 LTS
virtualbox: 6.1.36
```
```
$ uname -a
Linux daiuskedaisuke-VirtualBox 5.15.0-46-generic #49-Ubuntu SMP Thu Aug 4 18:03:25 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
```
## other
For android(armv8) build, the build fails because xdebug is enabled.
```
./compile.sh  -t android-aarch64 -x -j4 -f
[PocketMine] PHP compiler for Linux, MacOS and Android
[INFO] Checking dependencies
[opt] Set target to android-aarch64
[opt] Doing cross-compile
[opt] Set make threads to 2
[opt] Enabling abusive optimizations...
[opt] Will enable GD2
[INFO] Cross-compiling for Android ARMv8 (aarch64)
Warning:  OPcache cannot be used on static builds; this may have a negative effect on performance
./compile.sh: line 298: [: missing `]'
[PHP] downloading 8.0.22... done!
# ...
[PHP] enabling optimizations... checking... compiling... installing... generating php.ini... done!
  xdebug: downloading 3.1.5... done!
[xdebug] checking...
Error: Process completed with exit code 1.
```
```
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... configure: error: in `/home/daiuskedaisuke/pmmp/php-build-scripts/install_data/subdir/php/ext/xdebug':
configure: error: cannot run C compiled programs.
If you meant to cross compile, use `--host'.
See `config.log' for more details
```
https://github.com/DaisukeDaisuke/AndroidPHP/runs/7942855632?check_suite_focus=true
#### files
[install.log](https://github.com/pmmp/php-build-scripts/files/9391291/install.log)
[ext/xdebug/config.log](https://github.com/pmmp/php-build-scripts/files/9391330/config.log)

--- a/compile.sh
+++ b/compile.sh
@@ -295,7 +295,7 @@ if [ "$DO_STATIC" == "yes" ]; then
 	if [ "$FSANITIZE_OPTIONS" != "" ]; then
 		echo "[warning] Sanitizers cannot be used on static builds"
 	fi
-	if [ "$HAVE_XDEBUG" == "yes"]; then
+	if [ "$HAVE_XDEBUG" == "yes" ]; then
 	  write_out "warning" "Xdebug cannot be built in static mode"
 	  HAVE_XDEBUG="no"
 	fi

